pkg/components/velero/component.go

```diff
@@ -201,14 +201,14 @@ func (c *component) getProvider() (provider, error) {
 		providers = append(providers, c.Restic)
 	}
 
-	if len(providers) > 1 {
```

Comment:
It seems like something which should be committed separately. Why changing the order makes a difference?